src/git_test.rs

```diff
@@ -8,7 +10,10 @@ mod get_root_directory_path_tests {
     fn uses_git_rev_parse_top_level_command() {
         let exp = "/usr/me/foo";
         let target_dir = "";
-        let run_command = |cmd: &str, dir: Option<&str>, stream_io: bool| {
+        let run_command = |cmd: &str,
+                           dir: Option<&str>,
+                           stream_io: bool,
+                           _env: Option<&HashMap<String, String>>| {
```

Comment:
I forgot that we still haven't released the (correct) default block indent style on closures in a rustfmt 1.x version :sweat_smile: